I see that the continuous integration build failed with these errors:

```
05:58:35 Scanning dependencies of target phidgets_ik
05:58:35 [  9%] Building CXX object CMakeFiles/phidgets_ik.dir/src/ik_ros_i.cpp.o
05:58:36 In file included from /tmp/catkin_workspace/src/phidgets_drivers/phidgets_ik/src/ik_ros_i.cpp:1:0:
05:58:36 /tmp/catkin_workspace/src/phidgets_drivers/phidgets_ik/include/phidgets_ik/ik_ros_i.h:8:42: fatal error: phidgets_ik/SetDigitalOutput.h: No such file or directory
05:58:36 compilation terminated.
05:58:36 make[2]: *** [CMakeFiles/phidgets_ik.dir/src/ik_ros_i.cpp.o] Error 1
05:58:36 CMakeFiles/phidgets_ik.dir/build.make:62: recipe for target 'CMakeFiles/phidgets_ik.dir/src/ik_ros_i.cpp.o' failed
05:58:36 CMakeFiles/Makefile2:215: recipe for target 'CMakeFiles/phidgets_ik.dir/all' failed
05:58:36 make[1]: *** [CMakeFiles/phidgets_ik.dir/all] Error 2
05:58:36 Makefile:138: recipe for target 'all' failed
05:58:36 make: *** [all] Error 2
05:58:36 <== Failed to process package 'phidgets_ik': 
05:58:36   Command '['/tmp/catkin_workspace/install_isolated/env.sh', 'make', '-j1']' returned non-zero exit status 2
05:58:36 
05:58:36 Reproduce this error by running:
05:58:36 ==> cd /tmp/catkin_workspace/build_isolated/phidgets_ik && /tmp/catkin_workspace/install_isolated/env.sh make -j1
05:58:36 
05:58:36 Command failed, exiting.
05:58:36 # END SUBSECTION
```

This error does not appear on two computers where I compile this node. Could it be related to the configuration of the CI ?

--- a/phidgets_ik/CMakeLists.txt
+++ b/phidgets_ik/CMakeLists.txt
@@ -32,8 +32,8 @@ add_library(phidgets_ik src/ik_ros_i.cpp)
 
 add_executable(phidgets_ik_node src/phidgets_ik_node.cpp)
 
-add_dependencies(phidgets_ik ${catkin_EXPORTED_TARGETS})
-add_dependencies(phidgets_ik_node ${catkin_EXPORTED_TARGETS})
+add_dependencies(phidgets_ik ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
+add_dependencies(phidgets_ik_node ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
 
 target_link_libraries(phidgets_ik ${catkin_LIBRARIES})
 target_link_libraries(phidgets_ik_node ${catkin_LIBRARIES} phidgets_ik)

